R/new_matrixfit.R

```diff
@@ -376,8 +377,9 @@ new_matrixfit <- function(cf,
   }
 
   ## perform the bootstrap non-linear least-squares fit (NLS fit):
-  
+
   if (model == 'single') {
+    if(!missing(priors)) {stop('In the single model you must not specify priors!')}
```

Comment:
Why can't I put in priors for the single model? This seems like an arbitrary restriction.